modules/serverless-installing-cli-linux-rpm.adoc

```diff
@@ -0,0 +1,20 @@
+// Module is included in the following assemblies:
```

Comment:
@joaedwar : I think we can remove this file and its reference completely?  (as we have `modules/serverless-installing-cli-linux.adoc` )

Reply:
discussed with Joan, this file wont show up in the docs, but stays in the repo. Cool! We're good then.